### ISSUE TITLE
Add healthcheck endpoint

### ISSUE THREAD

**Issue by hbenali** [Meeds-io/meeds#2432]
**Is your feature request related to a problem? Please describe.**
This problem does not have any health check indicator
**Describe the solution you'd like**
Add an HTTP endpoint displaying the health status of the server application
**Describe alternatives you've considered**
The endpoint should not allocate user application sessions (Portal context, etc...)
It would be helpful for docker container health checks. Thanks

**Comment by srenault-meeds** [comment]
@rdenarie @boubaker 
I let you see this feedback as I am not expert of it.

**Comment by boubaker** [comment]
An endpoint like `/rest/private/management/pcontainer/Started` which returns 
![image](https://github.com/user-attachments/assets/91b6e84f-6f68-4f1a-a8ad-96addf42687f)
when the Container has successfully started ?

**Comment by hbenali** [comment]
Yes, something like this service. We would need an HTTP endpoint that does not allocate sessions for every health check and has no authentication (can we skip auth for requests coming from localhost ? eg: the docker container localhost). Thanks for the proposal.
